docs/support_files/world.md

```diff
@@ -1,32 +1,47 @@
 # World
 
-*World* is an isolated context for each scenario, exposed to the hooks and steps as `this`, enabling you to set and recall some state across the lifecycle of your scenario.
+*World* is an isolated context for each scenario, exposed to the hooks and steps as `this`, enabling you to set and recall some state across the lifecycle of your scenario. It is managed by a world class and each scenario is given an instance of the class when the test starts. You can use a default world that is part of Cucumber or create your own.
 
-Note that your hooks and step definition functions cannot reference the world as `this` if you use
-arrow functions. See [FAQ](../faq.md) for details.
+Note that your hooks and step definition functions cannot reference the world as `this` if you use arrow functions. See [FAQ](../faq.md) for details.
 
-A simple example:
+The simplest example is to relay values between steps.
 
 ```javascript
-const { When } = require('@cucumber/cucumber')
+const { Given, Then } = require('@cucumber/cucumber')
 
-When('something happens', async function() {
-  this.foo = 'bar'
+Given("my color is {string}", function(color) {
+  this.color = color
 })
+
+Then("my color should not be red", function() {
+  if (this.color === "red") {
+    throw new Error("Wrong Color");
+  }
+});
 ```
 
-As well as being able to have arbitrary state, you get some helpers preset on the World for you:
+With those step definitions in place
+
+```gherkin
+Scenario: Will pass
+  Given my color is "blue"
+  Then my color should not be red
+
+Scenario: Will fail
+  Given my color is "red"
+  Then my color should not be red  
+```
 
-* `this.attach`: function used for adding [attachments](./attachments.md) to hooks/steps
-* `this.log`: function used for [logging](./attachments.md#logging) information from hooks/steps
-* `this.parameters`: object of parameters passed in via the [CLI](../cli.md#world-parameters)
+## Cucumber World
 
-Some notes on the scope of World:
+The default world provided by Cucumber provides a number of formatting helpers that are passed into the constructor:
 
-- It's scoped to a single scenario only - not shared globally between scenarios. This reinforces a Cucumber principle: that scenarios should work entirely independently of one another.
-- If you're using [Retry](../retry.md), you'll get a fresh World for every attempt at your scenario, so state isn't retained between attempts.
```

Comment:
We've lost any reference to Retry now but this is an important detail.

Reply:
It's now back in in two locations - first paragraph and in the summary.

Reply:
Added two references in, one in the first paragraph and another in the summary.